### ISSUE TITLE
Texinfo failing to download on staging

### ISSUE THREAD

**Issue by r-burns** [NixOS/nixpkgs#139470]
Since https://github.com/NixOS/nixpkgs/pull/138063:

https://hydra.nixos.org/build/153907889/nixlog/3

```
error: unable to download 'https://ftpmirror.gnu.org/texinfo/texinfo-6.8.tar.xz': Couldn't resolve host name (6)
builder for '/nix/store/8x8fismhjkqwcl5qc2rq2zpm70sm9vv7-texinfo-6.8.tar.xz.drv' failed with exit code 1
```

cc @trofi

**Comment by trofi** [comment]
Fetch works locally here:

```
$ nix-prefetch-url https://ftpmirror.gnu.org/texinfo/texinfo-6.8.tar.xz
path is '/nix/store/ycj1q918l6852kndr75rlwmw43k5ybys-texinfo-6.8.tar.xz'
1i7yb7mrp3inz25zbzv2pllr4y7d58v818f1as7iz8mw53nm7dw
```

I wonder if it's related to `nix-daemon`'s bug in fetching new tarballs:
- https://github.com/NixOS/nix/issues/5089
- https://github.com/NixOS/nixpkgs/pull/137197

Tl;DR is: `nix-daemon` sometimes does not initialize glibc's NSS before entering chroot and fails to do it afterwards. https://github.com/NixOS/nixpkgs/pull/137197 will fix it but will also require builder update.

**Comment by vcunat** [comment]
Yes, it's certainly that nix bug.  I don't know... perhaps we can do the same as in the previous case: 28b6a91047e  I'm not aware of anything better **I** can do (personally) about this.

**Comment by trofi** [comment]
Would using `nixUnstable` instead of `nix` on hydra builder be an option until `nix` is fixed? (AFAIU `nixUnstable` already has the fix). Or too risky to get bugs creep in?

**Comment by vcunat** [comment]
It's unblocked now, though the underlying problem still isn't fixed.
